test/functionalTest/cases/1316_create_subscription/post_subs_expr_q_combined.test

```diff
@@ -0,0 +1,192 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Subscriptions V2 q expression (combined filter)
+
+--SHELL-INIT--
+dbInit CB
+# FIXME P10: replace with just 'brokerStart CB' once cache part gets implemented
+brokerStart CB 0 IPv4 -noCache
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Create subscription with q: temperature==20..40;status!=on
+# 02. Create entity with temperature=21 status=on (not notif)
+# 03. Update entity to temperature=19 status=off (not notif)
+# 04. Update entity to temperature=22 status=off (notif)
+# 05. Get accumulator dump (1 notifications)
+#
+
+echo "01. Create subscription with q: temperature==20..40;status!=on"
+echo "=============================================================="
+payload='
+{
+    "subject": {
+        "entities": [
+            {
+                "id": "E1",
+                "type": "T"
+            }
+        ],
+        "condition": {
+            "attributes": [ ],
+            "expression": {
+               "q": "temperature==20..40;status!=on"
+            }
+         }
+    },
+    "notification": {
+        "callback": "http://localhost:'$LISTENER_PORT'/notify",
+        "attributes": [ ]
+    },
+    "expires": "2050-04-05T14:00:00.00Z"
+}
+'
+orionCurl --url /v2/subscriptions --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Create entity with temperature=21 status=on (not notif)"
+echo "==========================================================="
+payload='{
+  "id": "E1",
+  "type": "T",
+  "temperature": 21,
+  "status": "on"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
```

Comment:
Semi-offtopic of this PR but... I wonder how (now that #1259 has landed in develop) this can work. In order to work, it should include `?options=keyValues`.

Maybe something is missing in #1259?